src/main/scala/org/camunda/feel/impl/interpreter/FeelInterpreter.scala

```diff
@@ -814,7 +725,7 @@ class FeelInterpreter {
     case ValDayTimeDuration(y) if (!y.isZero) =>
       withDayTimeDuration(x, x => ValNumber(x.toMillis / y.toMillis))
 
-    case _ => ValError(s"'$x / $y' is not allowed")
+    case _ => error(s"'$x / $y' is not allowed")
```

Comment:
❓ Does it make sense to introduce some sort of `INVALID_ARITHMETIC` type?

Reply:
Yes, we could introduce this new failure type here. Let's do this later. :rocket: